code/modules/power/pt_laser.dm

```diff
@@ -332,12 +332,11 @@
 		terminal.powernet.newload += amount
 
 /obj/machinery/power/pt_laser/proc/update_laser_power()
-	if(abs(output) > charge)
+	if(abs(src.output) > src.charge && src.firing)
```

Comment:
ambiguous

Reply:
`update_laser_power()` is called anytime `src.output` is changed. If the laser has already ceased firing there is no reason to call `stop_firing()` again for each adjustment of output that is still greater than the charge.

Reply:
I meant with parenthesis lol

Reply:
oh I added those too